src/demo-app/slide-toggle/slide-toggle-demo.scss

```diff
@@ -0,0 +1,8 @@
+.slide-toggle-example {
+  display:flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  align-content: flex-start;
+  align-items: flex-start;
```

Comment:
@DevVersion although just these properties set each slide toggle on it's own row, I kept the property `align-items: flex-start;` so the demo will not stretch the lines